### DISCUSSION TITLE
Change author

### DISCUSSION THREAD

**Discussion by cyclane** [nestdotland/nest.land#314] [Q&A] [answered]
Hello, is there a way to either:
- change the author of a module
- change my username
- publish a module from a new author

(or anything else that would result in the module having a different author)?

**Comment by maximousblk** [comment, marked as answer]
You can always create a new user and re-publish all the versions there.

Users can't change the author of the module.

What module are you trying to move?

**Reply by cyclane** [reply]
I've tried publishing the matrix.ts module from a new user with my intended name, but that gives an empty error. So the only way is to just publish it from a different user and with a slightly different name?

( https://nest.land/package/matrix.ts )

**Reply by maximousblk** [reply]
Yes, that is the only way of doing it.
We do not allow changing usernames and this will be even stricter when we migrate to scoped URLs
